flink-formats/flink-orc/src/main/java/org/apache/flink/orc/vector/OrcVectorizedBatch.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.orc.vector;
+
+/**
+ * Interface to unify orc reader in hive and orc reader without hive dependents.
+ */
+public interface OrcVectorizedBatch {
```

Comment:
I find this interface a little strange because it doesn't have any API to retrieve the vectorized batch, which means how this interface should be used depends on implementations. How about modify it like:
```
public interface OrcVectorizedBatchWrapper<T> {
    T getBatch();
    int size();
}
```

Reply:
It seems we don't need interface now.

Reply:
It has `size` method, should be interface.